README.md

```diff
@@ -31,6 +31,16 @@ Stampy UI is an interface for [stampy.ai](https://stampy.ai/read/Get_involved),
   (you need access to the doc of course, which you can request on the Discord in source code)
 - Replace the value for `{CODA_TOKEN}` in `wrangler.toml`
 
+4.1 Setup write access to the "Incoming questions" table in [Coda](https://coda.io/account)
+
+This step is only needed if you want to add new questions to Coda. Seeing as they will then
+go live, it would probably be better to only set this if you are working on that specific
+component, in order to not mess up the list of incoming questions.
+
+- When logged in to Coda, `Generate API token` in your Account settigns
+- Add restrictions: `Doc or table`, `Read and Write`, for the table with url `https://coda.io/d/_dfau7sl2hmG#_tugrid-S_6SYj6Tjm`
```

Comment:
this really only needs write access. 
I did separate tokens, so people without write access to coda can also develop stuff. Also because I didn't see a way to grant read access to everything, but write only to this one table